libdnf/transaction/Transformer.cpp

```diff
@@ -49,10 +49,34 @@ static const char *sql_create_tables =
 #include "sql/create_tables.sql"
     ;
 
+static const char * const sql_migrate_tables_1_2 =
+#include "sql/migrate_tables_1_2.sql"
+    ;
+
+const std::string Transformer::version = "1.2";
```

Comment:
Please do not expose variables directly as public API. Create a getter. 
eg.  `static const char * getVersion() noexcept { return "1.2"; };`